### ISSUE TITLE
Low accuracy on sample task

### ISSUE THREAD

**Issue by ndalton12** [graykode/xlnet-Pytorch#2]
Hi author,

Any idea why the performance accuracy is so low on the sample task you provide? 

I added the following lines to test accuracy:
`output = logits.transpose(1, 2)`
`predicts = F.softmax(output, dim=1).argmax(dim=1)`
`print((predicts == target).sum().item() / len(target))`

The accuracy for the task is ~0.01% after 100 epochs.

**Comment by graykode** [comment]
I think it is not important accuracy in this domain
